components/selection/selection-observer-mixin.js

```diff
@@ -16,7 +16,12 @@ export const SelectionObserverMixin = superclass => class extends superclass {
 			 * @type {object}
 			 */
 			selectionInfo: { type: Object },
-			_provider: { type: Object, attribute: false }
+			/**
+			 * HtmlElement of the `SelectionMixin` component this component wants to observe (alternative to selectionFor)
+			 * @type {string}
+			 */
+			selectionProvider: { attribute: false },
+			_provider: { state: true }
 		};
```

Comment:
I didn't add anything to the readme because there's already nothing there about this mixin...